pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -739,7 +737,6 @@ class Build {
                 // Archive the Mac and Windows pkg/msi
                 if (buildConfig.TARGET_OS == 'mac' || buildConfig.TARGET_OS == 'windows') {
                     try {
-                        context.sh 'for file in $(ls workspace/target/*.tar.gz workspace/target/*.pkg workspace/target/*.msi); do sha256sum "$file" > $file.sha256.txt ; done'
```

Comment:
By removing this part, installers for window and mac, wont get a *.sha256.txt file.
same as line 768